src/mods/jump.cpp

```diff
@@ -198,7 +271,17 @@ void tick() {
         }
     }
     if (enabled) {
-        jumping();
+        patch_minimap();
```

Comment:
Avoid patching every frame if possible

Reply:
Afaik the minimap needs to be patched whenever you enter the main mode MD_GAME or whenever jump mod is enabled and you're already in MD_GAME. Would there be an easy way to do this? Also do you remember what these two write_word commands do, I assume they disable the A-button minimap toggle?

Reply:
Doesn't seem too bad, I removed the patching on every frame